plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -97,8 +97,8 @@ def fetch_pods
             release_id: id
           }.to_kuber_selector
         }
```

Comment:
The same selector isn't going to work for getting pods and getting events.

Events aren't tagged with the labels that our pods or deployments are. To test this, try creating a Kubernetes deployment where one of the pods requires 10 gigs of RAM.  You can get the pods by filtering on project name, but not the events:

```
$ kubectl get pods -l "project=truth-service"
NAME                                            READY     STATUS    RESTARTS   AGE
truth-service-main-role-2819491091-secz1        0/1       Pending   0          2m
truth-service-secondary-role-2428193999-36sx9   1/1       Running   0          6d
truth-service-secondary-role-2428193999-8ydxk   1/1       Running   0          6d
$ kubectl get events -l "project=truth-service"
```

But here's one of the events about being unable to schedule the pod:

```
{
    "kind": "Event",
    "apiVersion": "v1",
    "metadata": {
        "name": "truth-service-main-role-2819491091-secz1.144b205219b6f108",
        "namespace": "jmoter",
        "selfLink": "/api/v1/namespaces/jmoter/events/truth-service-main-role-2819491091-secz1.144b205219b6f108",
        "uid": "0467039c-1158-11e6-84c0-0655266517f9",
        "resourceVersion": "843258",
        "creationTimestamp": "2016-05-03T17:54:01Z",
        "deletionTimestamp": "2016-05-03T18:54:29Z"
    },
    "involvedObject": {
        "kind": "Pod",
        "namespace": "jmoter",
        "name": "truth-service-main-role-2819491091-secz1",
        "uid": "02651e3b-1158-11e6-84c0-0655266517f9",
        "apiVersion": "v1",
        "resourceVersion": "843182"
    },
    "reason": "FailedScheduling",
    "message": "pod (truth-service-main-role-2819491091-secz1) failed to fit in any node\nfit failure on node (ip-10-0-243-157): Node didn't have enough resource: Memory, requested: 10485760000, used: 0, capacity: 7933714432\nfit failure on node (ip-10-0-253-136): Node didn't have enough resource: Memory, requested: 10485760000, used: 78643200, capacity: 7933714432\nfit failure on node (ip-10-0-248-157): Node didn't have enough resource: Memory, requested: 10485760000, used: 131072000, capacity: 7933714432\nfit failure on node (ip-10-0-246-137): Node didn't have enough resource: Memory, requested: 10485760000, used: 52428800, capacity: 7933714432\nfit failure on node (ip-10-0-245-34): Node didn't have enough resource: Memory, requested: 10485760000, used: 0, capacity: 7933714432\n",
    "source": {
        "component": "default-scheduler"
    },
    "firstTimestamp": "2016-05-03T17:54:01Z",
    "lastTimestamp": "2016-05-03T17:54:29Z",
    "count": 3,
    "type": "Warning"
}
```

Yes, this is a pain in the ass.